docs/howto/getting-started/enable-openstack-cli.md

```diff
@@ -0,0 +1,164 @@
+time, you can view all available OpenStack users by going to the 
+left-hand side pane on the {{extra.gui}} and selecting _Users_ > 
+_Openstack Users_.
+
+![All available OpenStack users](assets/ostack-cli/shot-06.png)
+
+## Creating and downloading an RC file
+
+On the {{extra.gui}} expand the left-hand side pane, click _Users_ and 
+then _Openstack Users_. You will then see, listed in the main pane 
+titled _OpenStack Users_, all available users. Click on the 
+three-dotted round icon on the right of the user you want to create an 
+RC file for. From the pop-up that appears, select _Generate RC file_.
+
+![Generate  RC file](assets/ostack-cli/shot-07.png)
+
+The RC file will be generated automatically. Before downloading it onto 
+your local computer, you must select one of the available projects to 
+relate the RC to. Do so and then click the blue _Download_ button.
+
+![Select project and download](assets/ostack-cli/shot-08.png)
+
+A "Save as" dialog window appears, specific to the Operating System you 
+are currently using. Select a convenient location and save your RC file.
+ 
+## Modifying and sourcing the RC file 
+
+The general naming for RC files goes like this:
+
+```plain
+your_username--region_name--project_name--rc
+```
+
+So, assuming your username is `olafsdottir`, and the RC file has been 
+created for the `fra1` region and the `katla` project, your RC file 
+name should be this:
+
+```plain
+olafsdottir--fra1--katla--rc
+```
+
+Take a look at the contents of this file --- they should be like this:
+
+```plain
+export OS_USERNAME=olafsdottir
+export OS_PASSWORD=<your password goes here>
+export OS_AUTH_URL=https://fra1.citycloud.com:5000
+export OS_USER_DOMAIN_NAME=...
+export OS_PROJECT_DOMAIN_NAME=...
+export OS_REGION_NAME=Fra1
+export OS_PROJECT_NAME="katla"
+export OS_TENANT_NAME="katla"
+export OS_AUTH_VERSION=3
+export OS_IDENTITY_API_VERSION=3
+```
+
+Before you source the RC, and thus initialize all relevant environment 
+variables, make sure to edit the file and put your OpenStack user 
+password in place of `<your password goes here>`. Then, go ahead and 
+source:
+
+```bash
+source olafsdottir--fra1--katla--rc
+```
+
+## Installing the OpenStack CLI tool
+
+If you do not have the OpenStack CLI tool readily available, use your 
+Operating System's package manager or `pip` to install it. Some 
+examples follow.
+
+=== "Ubuntu 22.04"
+	```bash
+	apt update && apt install python3-openstackclient 
+	```
+=== "Mac OS X with Homebrew"
+	```bash
+	brew install openstackclient
+	```
+=== "Python package"
+	```bash
+	pip install python-openstackclient
+	```
+
+## Testing access
+
+Provided you have already sourced your RC file, you can now use the 
+`openstack` command line tool to access various OpenStack APIs on the 
+{{extra.brand}} Cloud.
+
+To make sure your local installation of `openstack` works as expected, 
+type:
+
+```bash
+openstack token issue -f json
```

Comment:
Keep things as simple as possible; drop the `-f json` option.

Reply:
I added it because without it the output is hideously looking :) But I see your point.